extensions/wikia/AdEngine3/src/wad/bt-loader.js

```diff
@@ -0,0 +1,137 @@
+import { utils } from '@wikia/ad-engine';
+import { recInjector } from './rec-injector';
+
+const isDebug = utils.queryString.get('bt-rec-debug') === '1';
+const placementClass = 'bt-uid-tg';
+const placementsMap = {
```

Comment:
Just wonder if shouldn't we keep it in the context, along with slots definition. Below some kind of `rec` key. I.e. `slots.TOP_LEADERBOARD.rec.uid`, etc.

Reply:
Good suggestion. Moved to context, but the same way as bidders are.